### DISCUSSION TITLE
Identification of Out of Directive contracts in the ePO

### DISCUSSION THREAD

**Discussion by ignaciobavastro** [eprocurementontology/eprocurementontology#492] [Q&A]
@muricna 

For the calculation of SMS Indicators in the PPDS Project we need to identfy contracts that are Out of Directives.

According to [ISSUE 308](https://github.com/OP-TED/ted-rdf-mapping/issues/308) the Legal Basis Type “other-legal”, which flags when a Contract is Out of Directives will be added to EU Vocabularies on June 2023. This additional vocabulary value will affect all TED Contracts published after June 2023, or also all Contracts from previous years?
In case it only affects Contracts published after June 2023, could you please indicate if exists any workaround to detect if a Contract is Out of Directives using other ePO attributes, with the current ePO fields?

Thank you very much in advance!

**Comment by ignaciobavastro** [comment]
Moved this issue to the correct Board.

**Comment by muricna** [comment]
In the current standard forms the code other-legal comes from the authority table direct-award-justification.  Therefore it is only possible to see from the data if another legal basis was used in the case of procedures of type award of the contract without prior publication of a call for competition.  This code will not be used for eForms.
